pyphare/pyphare/pharein/simulation.py

```diff
@@ -175,6 +175,41 @@ def check_boundaries(dims, **kwargs):
 # ------------------------------------------------------------------------------
 
 
+# See: https://github.com/PHAREHUB/PHARE/wiki/exactSplitting
+# This should match possibleSimulators() in meta_utilities.h
+valid_refined_particle_nbr = {
+  # dim = {interp = [valid list refined_particle_nbr]}
+  1: {
+    1: [2, 3],
+    2: [2, 3, 4],
+    3: [2, 3, 4, 5]
+  },
+  2: {
+    1: [4, 8, 9],
+    2: [4, 8, 16],
+    3: [4, 8, 25]
```

Comment:
I think it would be nice to include `N=5`, as for this case, the pattern is not the same for all interp order : `PURPLE` for order 1, \PINK` for order 2 and 3.

Reply:
I'd rather add that next in a separate PR this one is already big enough I feel

https://github.com/PHAREHUB/PHARE/issues/229